lib/components/primitive-components/Group/Group.ts

```diff
@@ -140,6 +140,7 @@ export class Group<Props extends z.ZodType<any, any, any> = typeof groupProps>
             trace_clearance: props.autorouter.traceClearance,
           }
         : undefined,
+      anchor_alignment: props.pcbAnchorPosition ?? "center",
```

Comment:
Ugh what is happening
```suggestion
      anchor_alignment: props.pcbAnchorAlignment ?? null,
```